cmd/registry-server/main.go

```diff
@@ -99,7 +99,7 @@ func main() {
 		if err != nil {
 			bootLogger.WithError(err).Fatal("Failed to open config file")
 		}
-		// Expand environment variables before unmarshaling.
+		// Expand environment variables before unmarshalling.
```

Comment:
This repository uses both spellings roughly evenly, but there are more instances of the double-l spelling in user facing messages...